### PR TITLE
Feature/35653 - Qty updater

### PR DESCRIPTION
**Pull request by kjugi**
+ Add minus icon and list it in icon config
+ Add qty updater files
+ Link in styles

#107 

https://dev.snowdog.pro/issues/35653

### CODE REVIEW

File: components/03-modules/quantity-update/_quantity-update.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        text-align: center;
+
+        &:focus {
+            outline: none;
+        }
+    }
+
+    &__button {
+        border: 1px solid $gray-dark;
+        background: $white;
```
Comment (kruchy8) [inline components/03-modules/quantity-update/_quantity-update.scss]: I guess you should use some variables here...maybe those defined on line 7 & 8 :> ?

File: components/03-modules/quantity-update/_quantity-update.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    }
+
+    &__icon {
+        fill: $quantity-updater__icon-fill;
+        width: $quantity-updater__icon-spacing;
+        height: $quantity-updater__icon-spacing;
+    }
+
+    // Disable spin buttons in number input type
+    // Project rules
```
Comment (kruchy8) [inline components/03-modules/quantity-update/_quantity-update.scss]: Did you check it in IE ?

File: components/03-modules/quantity-update/_quantity-update.scss (hunk 1 above)
Comment (kjugi) [reply components/03-modules/quantity-update/_quantity-update.scss]: thanks! 😀

File: components/03-modules/quantity-update/_quantity-update.scss (hunk 2 above)
Comment (kjugi) [reply components/03-modules/quantity-update/_quantity-update.scss]: yes, checked and working fine

Comment (kruchy8) [reply components/03-modules/quantity-update/_quantity-update.scss]: great

**Pull request merged**